src/dom/global.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals window, document */
+
+/**
+ * @module utils/dom/global
+ */
+
+/**
+ * A helper (module) giving an access to the global `document` DOM object.
+ * Accessing this object using this helper allows easy and bulletproof
+ * testing, i.e. stubbing native properties:
+ *
+ *		import { document } from 'ckeditor5/utils/dom/global.js';
+ *
+ * 		// This stub will work for any code using global module in tests.
+ *		testUtils.sinon.stub( document, 'dir', 'rtl' );
+ */
+export { document };
```

Comment:
WAIT... what are you actually doing here? Exporting document as document... That doesn't make sense and I don't think it may actually work. Since you're exporting exactly the same object, then stubbing properties on it won't work (in case of window props).